generics/middleware/authenticator.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(req, res, next) {
   if (!req.rspObj) req.rspObj = {};
   var rspObj = req.rspObj;
   
-  if((req.path.includes("reports") || (req.query.csv && req.query.csv==true) ) && req.headers["internal-access-token"] === process.env.INTERNAL_ACCESS_TOKEN) {
+  if((req.path.includes("reports") || req.path.includes("programOperations") ) && req.headers["internal-access-token"] === process.env.INTERNAL_ACCESS_TOKEN) {
```

Comment:
ops report will take more time for huge data set so need to set more timeout for all the functionalities... and CSV is also not mandatory as well

Reply:
How much time for how many records ?

Reply:
5500 schools it will take 2 to 3 min.